### PR TITLE
Consular and Representative Split

### PR DESCRIPTION
**Pull request by Geevies**
* Consular Officer and Corporate Liaison has been split into two separate jobs with their own slots.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by fernerr — DISMISSED** [review]
Preemptive change request, as I need time to take a closer look at the mapping.

File: code/game/jobs/job/outsider/representative.dm [hunk 1]
```diff
@@ -85,8 +76,34 @@
 
 	return rep_objectives
 
+/datum/job/consular
```
Comment (Alberyk) [inline code/game/jobs/job/outsider/representative.dm:79]: This is missing the species restrictions.

File: code/game/jobs/job/outsider/representative.dm [hunk 2]
```diff
@@ -85,8 +76,34 @@
...
+	title = "Consular Officer"
+	flag = CONSULAR
+	department = "Civilian"
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "company officials"
```
Comment (Alberyk) [inline code/game/jobs/job/outsider/representative.dm:87]: This probably should be changed to something else.

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: code/game/jobs/job/outsider/representative.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/game/jobs/job/outsider/representative.dm:79]: how does this behave with contractors, btw? i assume as expected

Comment (Geevies) [reply code/game/jobs/job/outsider/representative.dm:79]: yeah, contractors cannot choose this option. you have to set your faction to unaffiliated to have this job as an option

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**